decider-Estimate/src/main/java/ca/on/oicr/pde/deciders/EstimateDecider.java

```diff
@@ -263,7 +263,11 @@ protected boolean checkFileDetails(ReturnValue returnValue, FileMetadata fm) {
                     }
                 }
             }
-            filteredMap.putIfAbsent(study, finalRVs);
+            if (finalRVs.size() > 1){ // check if we have more than one sample to launch workflow
+                filteredMap.putIfAbsent(study, finalRVs);
+            } else {
+                Log.stdoutWithTime("Will not launch workflows for sample size == 1");
```

Comment:
But the condition checks for size > 1, so it shouldn't say size==1, perhaps size < 2 ?

Reply:
So, this is a check to see if we have at least 2 samples to process.
ESTIMATE, ssGSEA programs fail to run for a matrix with only 1 sample